src/deploy/campaign/environments.ts

```diff
@@ -88,6 +92,14 @@ export const getConfig = async ({
     zeroVaultAddressConf = process.env.ZERO_VAULT_ADDRESS;
   }
 
+  process.env.MOCK_Z_TOKEN === "true";
```

Comment:
My comment before, which I see you used below as `!!process.env.MOCK_Z_TOKEN`, was that because you read this value twice, it would be be better to just store it in a variable to avoid that. 

Also, you only changed the line I left the comment on. Here we still do `if (A === true)` which is not needed.

Only a small nitpick, but good code is comprised of many small habits :)

Reply:
> Also, you only changed the line I left the comment on. Here we still do if (A === true) which is not needed.

Did understand what do you mean. I replaced this construction ( if (A === true) ) with simple if (A) and don't see any other.